cc-pk-service/openapi.yaml

```diff
@@ -23,7 +20,7 @@ paths:
             text/plain:
               schema:
                 type: string
-  '/v1/items/{userId}':
+  '/v1/users/{userId}/items':
```

Comment:
Will this also affect the next path in the future? Meaning '/v1/items/{itemId}' -> '/v1/users/{userId}/items/itemId' or something?

Reply:
Aa, already found out. So that path is unchanged. Looks good!